src/xform_to_datalog/authorization_logic_test.cc

```diff
@@ -21,35 +21,12 @@
 #include <vector>
 
 #include "src/common/testing/gtest.h"
+#include "src/xform_to_datalog/authorization_logic_test.h"
 
 namespace fs = std::filesystem;
```

Comment:
When declaring namespace aliases, we're supposed to always place them in the innermost namespace (so inside `namespace raksha::xform_to_datalog {` here).